rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -227,6 +227,38 @@ void debugger_frame::keyPressEvent(QKeyEvent* event)
 	const auto cpu = this->cpu.lock();
 	int i = m_debugger_list->currentRow();
 
+	switch (event->key())
+	{
+	case Qt::Key_F1:
+	{
+		QDialog* diag = new QDialog(this);
```

Comment:
why do you always call it diag..... this makes zero sense

Reply:
Fixed, I fixed it in other places as well.